lib/language_pack/ruby_version.rb

```diff
@@ -38,6 +38,10 @@ def initialize(bundler_output, app = {})
       @version_without_patchlevel = @version.sub(/-p-?\d+/, '')
     end
 
+    def ruby_192_or_lower?
+      Gem::Version.new(self.ruby_version) <= Gem::Version.new("1.9.2")
+    end
```

Comment:
Is it worth just inlining this since we only use it once?

`ruby.version <= Gem::Version.new("1.9.2")`

Reply:
Here's a mockup:

```ruby
      # Need to remove `./bin` folder since it links to the wrong --prefix ruby binstubs breaking require in Ruby 1.9.2 and 1.8.7.
      # Because for 1.9.2 and 1.8.7 there is a "build" ruby and a non-"build" Ruby
      paths << "#{File.expand_path(".")}/bin" unless Gem::Version.new(ruby_version.ruby_version) <= Gem::Version.new("1.9.2")
```

I don't prefer it.